custom_components/tuya_local/devices/srne_energy_storage_battery.yaml

```diff
@@ -0,0 +1,310 @@
+name: LiFP battery
+products:
+  - id: l2ak4zqufuarm0j3
+    manufacturer : SRNE
+    model: SR-SE05B
+entities:
+  - entity: sensor
+    class: battery
+    translation_key: Charge
+    dps:
+      - id: 103
+        type: integer
+        name: sensor
+        unit: "%"
+        class: measurement
+  - entity: sensor
+    class: voltage
+    translation_key: Battery voltage
+    dps:
+      - id: 102
+        type: integer
+        name: sensor
+        unit: V
+        class: measurement
+        precision: 2
+        mapping:
+          - scale: 100
+  - entity: sensor
+    class: current
+    name: Charge current
+    dps:
+      - id: 101
+        type: integer
+        name: sensor
+        unit: A
+        class: measurement
+        mapping:
+          - scale: 100
+  - entity: sensor
+    name: Cycle number
+    dps:
+      - id: 108
+        type: integer
+        name: sensor
+        unit: cyc
```

Comment:
I don't think this is a valid unit in any system, either Hz if this is frequency, or spell it out in full as `cycles`